crates/wast/src/component/alias.rs

```diff
@@ -36,25 +36,64 @@ pub struct CoreAlias<'a> {
     pub target: CoreAliasTarget<'a>,
 }
 
+impl<'a> CoreAlias<'a> {
+    /// Parses only an outer type alias.
+    pub fn parse_outer_type_alias(parser: Parser<'a>) -> Result<Self> {
```

Comment:
I think this may be dead code at this point? If not though could this be deduplicated with the implementation of `Parse` below?